Src/Managers/font_manager/font_manager.py

```
@@ -0,0 +1,90 @@
+        config = json.load(path_config.open())
+        self.fonts = defaultdict(defaultdict)
+        self.default = None
+        for font_name, font_config in config.items():
+
+            font_ids = [dpg.generate_uuid() for _ in font_config['sizes']]
+
+            font = FontUnit(font_config['path'], font_config['hints'],
+                            font_name, font_config['sizes'][0], font_ids[0])
+            self.fonts[font_name][font_config['sizes'][0]] = font
+
+            for size, curr_id, next_id in zip(font_config['sizes'][1:], font_ids[:-1], font_ids[1:]):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)